turf/circle/_circle.py

```diff
@@ -0,0 +1,60 @@
+from functools import reduce
+from typing import TypeVar, Dict, List
+
+from turf import destination, polygon, Polygon
+from turf.invariant import get_coords_from_features
+from turf.utils.error_codes import error_code_messages
+from turf.utils.exceptions import InvalidInput
+
+
+Center = TypeVar("Center", List, Dict, Polygon)
+Radius = TypeVar("Radius", int, float)
+
+
+def add_coordinates(coords, center, radius, bearing, options):
```

Comment:
Should this be part of the helpers module or is this method specific for the circle tool?

Reply:
This is mostly a helper method for the reduce on the main `circle` function, so I believe it belongs here.

Reply:
Thanks for the explanation.